projects/extension/src/background/ConnectionManager.ts

```diff
@@ -52,6 +53,14 @@ export interface ToConnectionManagerAddWellKnownChain {
   databaseContent?: string
 }
 
+export interface ToConnectionManagerExtConfig {
+  origin: "trusted-user"
+  type: "ext-config"
+  chainId: string
+  chainName: string
+  jsonRpcMessage: string
+}
```

Comment:
This is exactly the same thing as the regular RPC message.
The `ConnectionManagerWithHealth` already has a system in place to send RPC requests. There's no need to add *another* mechanism to the `ConnectionManager`. Only the `ConnectionManagerWithHealth` needs to be changed.